src/main/menu/actions/file.js

```diff
@@ -133,6 +133,8 @@ const handleResponseForSave = async (e, { id, filename, markdown, pathname, opti
   }
 
   filePath = path.resolve(filePath)
+  const extension = path.extname(filePath) || '.md'
+  filePath = !filePath.endsWith(extension) ? filePath += extension : filePath
```

Comment:
Thanks for you suggestion but why should we manipulate the user selection here? The user choose this filename a few lines above.

Reply:
Maybe you don't understand this problem. It's obviously a bug, not the use problem you said. This causes different windows to be opened for the same file with different file names.

Reply:
`.md` is required as fallback becasue `writeMarkdownFile` use this extension as fallback. Both `menu/actions` and `filesystem` should be refactored soon.